extensions/io/org.eclipse.smarthome.io.mqttembeddedbroker/META-INF/MANIFEST.MF

```diff
@@ -20,6 +20,7 @@ Import-Package:
  org.eclipse.smarthome.io.transport.mqtt.reconnect,
  org.eclipse.smarthome.io.transport.mqtt.sslcontext,
  org.osgi.framework,
+ org.osgi.service.cm;version="1.5.0",
```

Comment:
please remove the version restriction.

Reply:
Can I teach Eclipse not to add these automatically? Would be awesome